app/keepers/keepers.go

```diff
@@ -447,11 +444,8 @@ func (appKeepers *AppKeepers) InitKeepers(
 
 	var icaControllerStack porttypes.IBCModule
 	icaControllerStack = interchainstaking.NewIBCModule(appKeepers.InterchainstakingKeeper)
-	interchainstakingIBCModule, ok := icaControllerStack.(interchainstaking.IBCModule)
-	if !ok {
-		panic("unable to create interchainstaking module from ICA controller stack")
```

Comment:
why remove this panic? I'd rather panic here with a clear error than potentially randomly later with some obscure stack trace because we didn't bail early.

Reply:
Im updated it by [this line](https://github.com/notional-labs/quicksilver/blob/950d879a0ef010866226d94c09b3f73e7ce5adde/app/keepers/keepers.go#L447), because we have define it [above](https://github.com/notional-labs/quicksilver/blob/950d879a0ef010866226d94c09b3f73e7ce5adde/app/keepers/keepers.go#L130) that